packages/@lwc/aria-reflection/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { AriaPropNameToAttrNameMap, keys } from '@lwc/shared';
+import { detect } from './detect';
+import { patch } from './polyfill';
+
+export function applyAriaReflection(prototype: any = Element.prototype) {
```

Comment:
I've started avoiding default exports because they tend to cause problems with CommonJS compatibility. There are some other good reasons outlined here: https://basarat.gitbook.io/typescript/main-1/defaultisbad

Reply:
Another reason I'll add: it makes it easy to introduce new APIs without breaking backwards compat. Just rename the function and export the new function while keeping the old one.